### PR TITLE
content/en/blog/2021: add "Announcing etcd 3.5"

### PR DESCRIPTION
**Pull request by gyuho**
/cc @xiang90 @ptabor @hexfusion 

ref. https://github.com/etcd-io/etcd/issues/12330
ref. https://github.com/etcd-io/etcd/issues/12970

### CODE REVIEW

File: content/en/blog/2021/announcing-etcd-3.5.md [hunk 1]
```diff
@@ -0,0 +1,126 @@
+---
+title: Announcing etcd 3.5
+date: 2021-06-15
```
Comment (gyuho) [inline content/en/blog/2021/announcing-etcd-3.5.md:3]: @hexfusion @ptabor Do we have any release date in mind? (We can change it later as well)

**Comment by chalin** [comment]
@gyuho - could you rebase? That will allow us to see the blog post under the Netlify preview (following #325). Thanks.

**Comment by gyuho** [comment]
@chalin Done. Thanks!

**Comment by chalin** [comment]
@gyuho - I submitted a commit with fixes to the links, and added an `authors` field

The page preview now works: https://deploy-preview-312--etcd.netlify.app/blog/2021/announcing-etcd-3.5/

/cc @nate-double-u

**Comment by gyuho** [comment]
Preview looks good to me. Thanks!

**Comment by chalin** [comment]
@gyuho - I propose that we merge this PR. I'll make a pass and submit copyedits in a followup PR. How does that sound?

**Review by chalin — APPROVED** [review]
Approved to merge as a draft page. As I mentioned in the previous comment, I'll review it after from and submit followup PRs.

**Comment by xiang90** [comment]
lgtm

File: content/en/blog/2021/announcing-etcd-3.5.md (hunk 1 above)
Comment (hexfusion) [reply content/en/blog/2021/announcing-etcd-3.5.md:5]: 6/15 seems reasonable given the current trajectory +1

**Comment by gyuho** [comment]
> I propose that we merge this PR. I'll make a pass and submit copyedits in a followup PR. How does that sound?

Sounds good. Let's merge then. Thanks!

**Comment by chalin** [comment]
@hexfusion @xiang90 et all: for followup edits, see #331

**Review by tangcong — COMMENTED** [review]
same behavior as 3.3? @gyuho

File: content/en/blog/2021/announcing-etcd-3.5.md [hunk 2]
```diff
@@ -0,0 +1,380 @@
...
+etcd 3.4 release made backend read transactions fully concurrent by copying
+transaction buffers rather than sharing between writes and concurrent reads (see
+[code change from 3.4 release](https://github.com/etcd-io/etcd/pull/10523)).
+However, such buffering mechanism comes with unavoidable copy overhead and
+negatively impacted write-heavy transaction performance, as creating concurrent
+read transactions acquires a mutex lock which then blocks incoming write
+transactions. etcd 3.5 made multiple improvements to further increase the
+transaction concurrency: (1) If a transaction includes a PUT(update) operation,
+the transaction instead shares the transaction buffer between reads and writes
+(same behavior as 3.4) in order to avoid copying buffers. This transaction mode
```
Comment (tangcong) [inline content/en/blog/2021/announcing-etcd-3.5.md:165]: same behavior as 3.3?

File: content/en/blog/2021/announcing-etcd-3.5.md [hunk 3]
```diff
@@ -0,0 +1,380 @@
...
+had redundant backend database open operations and as a result, reloading
+40-million keys took over 5-minute, and the fix reduced the restart time by half
+-- [tangcong@'s fix (from Tencent)](https://github.com/etcd-io/etcd/pull/11779),
+(4) if etcd crashed before completing defragment, the next defragment operation
+might have read the corrupted file, and the fix was to ignore and overwrite the
+existing file -- see [jpbetz@'s fix (from
+Google)](https://github.com/etcd-io/etcd/pull/11613), (5) client cancelling
+watch does not signal server creating leaky watchers, and the fix was to
+explicitly send a cancel request to the server -- see [jackkleeman@'s fix (from
+Apple)](https://github.com/etcd-io/etcd/pull/11613).
```
Comment (tangcong) [inline content/en/blog/2021/announcing-etcd-3.5.md:119]: https://github.com/etcd-io/etcd/pull/11613 -> https://github.com/etcd-io/etcd/pull/11850

Comment (tangcong) [reply content/en/blog/2021/announcing-etcd-3.5.md:119]: i have submitted pr #384 to fix the above two typo issues.

**Pull request merged**